package/Rakefile

```
@@ -176,6 +178,117 @@ class MilterManagerTask < PackagesGroongaOrgPackageTask
     end
     task :source => source_tasks.collect {|task| "source:#{task}"}
   end
+
+  def apply_obs_pkg_metadata(project_name, package_name)
+    Dir.chdir(project_name) do
+      require 'tempfile'
+      Tempfile.create("meta.xml") do |f|
+        f.puts(<<-EOS
+<package name="#{package_name}" project="#{project_name}">
+  <title>Milter manager #{detect_version}</title>
+  <description/>
+</package>
+EOS
+              )
```

We can use `CGI.escape_html` with `require "cgi/util"`.

```suggestion
        meta_xml.puts(<<-XML)
<package name="#{CGI.escape_html(package_name)}" project="#{CGI.escape_html(project_name)}">
  <title>milter manager #{CGI.escape_html(detect_version)}</title>
  <description/>
</package>
        XML
```